config/flags/flags.go

```diff
@@ -0,0 +1,5 @@
+package flags
+
+import "sync/atomic"
+
+var EnableExperimentalVerification atomic.Bool
```

Comment:
**Thought**: I wonder if it makes sense to move more of the cli flags to this package. Making that change would be beyond the scope of this PR but curious to hear your thoughts.

On the other hand `EnableExperimentalVerification` could just be passed as an argument to the `print` functions

Reply:
> **Thought**: I wonder if it makes sense to move more of the cli flags to this package. Making that change would be beyond the scope of this PR but curious to hear your thoughts.

We'd have to review how flags are being set/accessed to see if it would improve the code quality.

There were a few places where it wasn't possible/practical to pass `d.EnableExperimentalVerification` (either ugly code or cyclical references). Most of those cases no longer exists, however, this still seems like the cleanest way.